crates/red_knot_python_semantic/src/types.rs

```diff
@@ -2352,18 +2445,42 @@ pub enum Truthiness {
 }
 
 impl Truthiness {
+    pub(crate) const fn from_bool(value: bool) -> Self {
+        if value {
+            Self::AlwaysTrue
+        } else {
+            Self::AlwaysFalse
+        }
+    }
```

Comment:
hmm, do we need this, considering we already implement `From<bool>` for Truthiness?